lesson26/tailwind.config.cjs

```diff
@@ -13,12 +13,28 @@ module.exports = {
         "Hiragino Sans",
         "Meiryo",
         "sans-serif"
+      ],
+      indexBody: [
+        "YuGothic",
+        "Yu Gothic",
+        "Yu Gothic UI",
+        "ＭＳ ゴシック",
+        "ヒラギノ角ゴシック",
+        "Hiragino Sans",
+        "Hiragino Kaku Gothic ProN",
+        "ヒラギノ角ゴ ProN W3",
+        "sans-serif"
       ]
     },
-    extend: {},
+    extend: {
+      transitionDelay: {
+        '2000': '2000ms',
+      }
+    },
   },
   plugins: [
-    plugin(function ({ addBase, addComponents }) {
+    plugin(function ({ addBase, addComponents,addVariant }) {
+      addVariant('active', '&.is-active'),
```

Comment:
This is the Variant to use when the is-active class is attached. 
You can add your own Variant with addVariant.

ref:https://tailwindcss.com/docs/plugins#adding-variants